server/neptune/gateway/event/preworkflow/hook_runner.go

```diff
@@ -0,0 +1,54 @@
+package preworkflow
+
+import (
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/neptune/gateway/event/source"
+)
+
+type HooksRunner interface {
```

Comment:
Same as above, should we make it private and move this to the file where this interface is being used?

Reply:
removed interface for now (until the user is added in a separate PR)